docs/development/testguide.rst

```diff
@@ -122,6 +122,11 @@ Enable PEP8 compliance testing with ``pep8=True`` in the call to
     variable (see :doc:`/config/index`) or the `pytest`_ method described
     above.
 
+Astropy Test Function
+#####################
```

Comment:
This needs to be ``^`` instead of ``#``, because Sphinx needs to have consistent section headings or else it spurts out all kinds of warnings.